src/components/IDCard.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+function IDCard(props) {
+    const { lastName, firstName, gender, height, birth, picture} = props;
+
+  return (
+    <div className='container'>
+        <div>
+            <img src={picture} alt="user picture" />
+        </div>
+        <div>
+            <p><span className='bold-text'>First Name: </span>{lastName}</p>
+            <p><span className='bold-text'>Last Name: </span> {firstName}</p>
+            <p><span className='bold-text'>Gender: </span>{gender}</p>
+            <p><span className='bold-text'>Height: </span>{height}cm</p>
+            <p><span className='bold-text'>Birth: </span>{birth.toLocaleString()}</p> 
```

Comment:
Here, to not display the hours, you would need to use slice, like:

` <p><span className='bold-text'>Birth: </span>{birth.toLocaleString().slice(0,10)}</p> `